frontend/src/app/result-list/result-list.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Button, Result as ResultContent} from 'antd';
+import { inject, observer } from 'mobx-react';
+
+import { ResultStore, Result } from '../../stores/resultStore';
+import { RESULT_LIST_MAX_SIZE } from '../../App.constants';
+import { ResultListEntry } from './';
+
+interface IProps {
+  resultStore?: ResultStore;
+}
+
+@inject('resultStore')
+@observer
+export class ResultList extends React.Component<IProps> {
+  public render() {
+    const results = this.props.resultStore.dummyResults;
+    // const results: Result[] = [];
```

Comment:
try uncommenting this line and commenting line 17 above it to see what the page now looks like!